app/uk/gov/hmrc/economiccrimelevyregistration/navigation/OtherEntityTypePageNavigator.scala

```diff
@@ -16,24 +16,19 @@
 
 package uk.gov.hmrc.economiccrimelevyregistration.navigation
 
-import play.api.mvc.{Call, RequestHeader}
+import play.api.mvc.Call
 import uk.gov.hmrc.economiccrimelevyregistration.controllers.routes
-import uk.gov.hmrc.economiccrimelevyregistration.models.{CheckMode, Mode, NormalMode, Registration}
+import uk.gov.hmrc.economiccrimelevyregistration.models.{NormalMode, Registration}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendHeaderCarrierProvider
 
-import scala.concurrent.Future
+class OtherEntityTypePageNavigator extends PageNavigator with FrontendHeaderCarrierProvider {
```

Comment:
You don't need the `FrontendHeaderCarrierProvider` when it's not an `AsyncPageNavigator`

Reply:
Removed.